client/lib/user/user.js

```diff
@@ -231,7 +236,7 @@ User.prototype.clear = function() {
 	delete this.settings;
 	store.clear();
 	if ( config.isEnabled( 'persist-redux' ) ) {
-		localforage.removeItem( 'redux-state' );
+		localforage.removeItem( 'redux-state', onClear );
```

Comment:
Good catch, what do we want to do if we're unable to clear state? Probably redirect anyway?

```jsx
localForage.removeItem( 'redux-state' ).then( onClear ).catch( onClear );
```